onos-topo/values.yaml

```diff
@@ -22,7 +22,10 @@ store:
   controller: atomix-controller.kube-system.svc.cluster.local:5679
   raft:
     partitions: 1
-    partitionSize: 1
+    clusters: 1
+    replicas: 1
+    backend:
+      image: atomix/raft-replica:latest
```

Comment:
Not sure about this image yet. It should be confirmed by Jordan.

Reply:
It is correct.